independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/update/rewrite/QuarkusUpdateCommand.java

```diff
@@ -143,6 +154,7 @@ private static List<String> getMavenProcessSourcesCommand(String mvnBinary) {
         List<String> command = new ArrayList<>();
         command.add(mvnBinary);
         command.add("-B");
+        command.add("-X");
```

Comment:
`-X` can be super noisy (I know it as I have been debugging a plugin for last 2 weeks :disappointed: [not for this issue]), is this on activated on demand?